specification/resource/semantic_conventions/container.md

```diff
@@ -14,4 +14,8 @@
 | `container.runtime` | string | The container runtime managing this container. | `docker`; `containerd`; `rkt` | Recommended |
 | `container.image.name` | string | Name of the image the container was built on. | `gcr.io/opentelemetry/operator` | Recommended |
 | `container.image.tag` | string | Container image tag. | `0.1` | Recommended |
+| `container.image.id` | string | The ID of the container image. Usually a hash algorithm followed by a UUID. The UUID might be abbreviated. | `sha256:f90d814248a0` | Recommended |
+| `container.command` | string | The command used to run the container (i.e. the command name). | `otelcontribcol` | Recommended |
+| `container.command_line` | string | The full command run by the container as a single string representing the full command. It is recommended to remove embedded credentials or sensitive data to prevent potential leakage. | `otelcontribcol --config config.yaml` | Recommended |
+| `container.command_args` | string[] | All the command arguments (including the command/executable itself) run by the container. It is recommended to remove embedded credentials or sensitive data to prevent potential leakage. | `[otelcontribcol, --config, config.yaml]` | Recommended |
```

Comment:
Given the recommendation to remove credentials + sensitive data, I'd definitely suggest AGAINST these being recommended, and possibly instead make them "Opt-In"

Reply:
In the [process resource semconv](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/process.md#process) it has conditionally required logic. Given the similarities here, it may make sense to align with that as closely as possible.

Reply:
I set them as conditionally required following your suggestions.

Reply:
I think also these should be Opt-In (also the process ones). I opened a separated issue to discuss it https://github.com/open-telemetry/semantic-conventions/issues/129